sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -671,6 +672,11 @@ export enum DataType {
     String = "String"
 }
 
+// @public
+export interface DedicatedGatewayRequestOptions {
+    maxIntegratedCacheStaleness?: number;
```

Comment:
if this is in time, we usually suffix the units (e.g. `maxIntegratedCacheStalenessInMs`)